kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -169,6 +169,14 @@
   public static final ConfigDef.Range PRODUCE_GRACE_PERIOD_MS_VALIDATOR =
       ConfigDef.Range.between(0, Integer.MAX_VALUE);
 
+  public static final String PRODUCE_EXECUTOR_THREAD_POOL_SIZE =
+      "api.v3.produce.executor.thread.pool.size";
```

Comment:
nit: Maybe also include "response" in the name here? As this pool will be used mostly for blocking tasks, it won't be easy to also make use of it for other tasks in the Produce pipeline.

Reply:
`api.v3.rest.callback.thread.pool.size` see my comment above.  Let me know if you're not keen on that !

Reply:
FWIW I prefer what you originally had, but with `response` replacing `executor`.